src/pages/validator/amplifier/verifier-onboarding.mdx

```diff
@@ -104,6 +106,9 @@ docker run axelarnet/axelar-ampd:v0.6.0 verifier-address
 </tab-item>
 
 </tabs>
+<Callout>
+The address for your verifier node will also be the address rewards are sent to (this will change in the future). You should make sure to claim and move any available rewards regularly to mitigate risks associated with this.
```

Comment:
```suggestion
The address for your verifier node will also be the address that rewards are sent to (this will change in the future). You should make sure to claim and move any available rewards regularly to mitigate any associated risks.
```